inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/decisions/AdvancedDecision.java

```diff
@@ -0,0 +1,111 @@
+package uk.ac.ucl.rits.inform.informdb.decisions;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import uk.ac.ucl.rits.inform.informdb.TemporalCore;
+import uk.ac.ucl.rits.inform.informdb.annotation.AuditTable;
+import uk.ac.ucl.rits.inform.informdb.identity.HospitalVisit;
+import uk.ac.ucl.rits.inform.informdb.identity.Mrn;
+
+import javax.persistence.Id;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import java.time.Instant;
+
+/**
+ * Holds information relevant to advanced decisions taken by patients.
+ */
+@Entity
+@Data
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+@AuditTable
+public class AdvancedDecision extends TemporalCore<AdvancedDecision, AdvancedDecisionAudit> {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    /** Identifier for specific advanced decision. */
+    private long advancedDecisionId;
+
+    @ManyToOne
+    @JoinColumn(name = "advancedDecisionTypeId", nullable = false)
+    /** Identifier for type of specific advanced decision. */
+    private AdvancedDecisionType advancedDecisionTypeId;
+
+    @ManyToOne
+    @JoinColumn(name = "hospitalVisitId", nullable = false)
+    /** Identifier for hospital visit of specific advanced decision. */
+    private HospitalVisit hospitalVisitId;
+
+    @ManyToOne
+    @JoinColumn(name = "mrnId", nullable = false)
+    /** Identifier for patient of specific advanced decision. */
+    private Mrn mrnId;
+
+    @Column(nullable = false)
+    /**
+     * Identifier assigned in source system that should be unique across all advanced decisions recorded in the
+     * hospital.
+     */
+    private Long advancedDecisionNumber;
+
+    /** Optional fields for advanced decisions. */
+    /** Indicates whether (True) or not (False) an advanced decision ceased to exist based on patient discharge.*/
+    private Boolean closedDueToDischarge = false;
+
+    /** Time when the information for the advanced decision has been last updated. */
+    private Instant statusChangeTime;
+
+    /** Time when the advanced decision for the patient's hospital visit has been recorded first. */
+    private Instant requestedDateTime;
```

Comment:
this is a tricky one, requested time could be when the consult was requested for (e.g. can I have a frailty consult tomorrow at 10am) or it could be the time the request was made. might be worth working out a more clear name. 

scheduled datetime seems reasonable for the time when you're asking the consult to happen
requesting datetime might be a bit more clear? but not sure

Reply:
I think we settled on using Datetime as a naming convention

```suggestion
    private Instant requestedDatetime;
```